debian/rules

```diff
@@ -1,103 +1,18 @@
-#!/usr/bin/make -f
+#! /usr/bin/make -f
 
-dist_release := $(shell lsb_release -cs)
-
-dh_extra_flags = -plandscape-common -plandscape-client
-
-include /usr/share/python3/python.mk
-ifeq (,$(py_sitename))
-	py_sitename = site-packages
-	py_libdir = /usr/lib/python$(subst python,,$(1))/site-packages
-	py_sitename_sh = $(py_sitename)
-	py_libdir_sh = $(py_libdir)
-endif
+export DH_ALWAYS_EXCLUDE=.git
+export DH_VERBOSE = 1
 
 package = landscape-client
 root_dir = debian/tmp
 SHAREDIR = /usr/share/landscape
 LIBDIR = /usr/lib/landscape
 
-revision = $(shell dpkg-parsechangelog | grep ^Version | cut -f 2 -d " "| cut -f 2 -d "-")
-
-landscape_common_substvars = debian/landscape-common.substvars
-landscape_client_substvars = debian/landscape-client.substvars
-
-build-arch: build
+%:
+	dh $@ --with python3 --buildsystem=pybuild
 
-build-indep: build
-
-build: build-stamp
-build-stamp:
-	dh_testdir
-	sed -i -e "s/^DEBIAN_REVISION = \"\"/DEBIAN_REVISION = \"-$(revision)\"/g" landscape/__init__.py
-	python3 setup.py build
+override_dh_auto_install:
+	dh_auto_install
 	make -C apt-update
-	touch build-stamp
-
-clean:
-	dh_testdir
-	dh_testroot
-	rm -f build-stamp
-	rm -rf build
-	make -C apt-update clean
-	dh_clean
-	sed -i -e "s/^DEBIAN_REVISION = .*/DEBIAN_REVISION = \"\"/g" landscape/__init__.py
-
-install: build
-	dh_testdir
-	dh_testroot
-	dh_prep
-	dh_installdirs
-
-	python3 setup.py install --root $(root_dir)/ $(py_setup_install_args)
-
-	# Keep in mind that some installed files are defined in setup.py
-	install -D -o root -g root -m 755 debian/landscape-sysinfo.wrapper $(root_dir)$(SHAREDIR)/landscape-sysinfo.wrapper
-	install -D -o root -g root -m 755 apt-update/apt-update $(root_dir)$(LIBDIR)/apt-update
-
-binary-indep:
-# do nothing
-#
-binary-arch: build install
-	dh_lintian
-	dh_testdir
-	dh_testroot
-	dh_installdocs
-	dh_installman
-	dh_installchangelogs
-	dh_install --sourcedir debian/tmp/
-	dh_systemd_enable --no-enable
```

Comment:
I'm not sure dh does what we want by default here. We used to not enable the service by default; only when landscape-config wrote the client configuration. We probably want to keep that to avoid the unit failing after install.

Reply:
The only valuable item to keep here is the `dh_systemd_enable --no-enable`

I'll push an updated commit to include it.